pkg/yurtctl/constants/constants.go

```diff
@@ -186,7 +186,7 @@ spec:
         - /bin/sh
         - -c
         args:
-        - "nsenter -t 1 -m -u -n -i -- /bin/yurtctl convert edgenode --yurthub-image {{.yurthub_image}} --join-token {{.joinToken}}"
+        - "cp /usr/local/bin/yurtctl /tmp && nsenter -t 1 -m -u -n -i -- /var/tmp/yurtctl convert edgenode --yurthub-image {{.yurthub_image}} --join-token {{.joinToken}} && rm /tmp/yurtctl"
```

Comment:
can we execute the `/usr/local/bin/yurtctl` directly?

Reply:
The operations need to modify the files on the Convert/RevertServant Pod host, so we need to mount yurtctl to the host and run it in the host environment. If run yurtctl directly in the container, the content of the file in the pod is modified.

Reply:
That makes sense. Thanks for explaining